spot-client/src/spot-tv/ui/views/home.js

```diff
@@ -336,6 +331,16 @@ export class Home extends React.Component {
             </div>
         );
     }
+
+    /**
+     * Generates the URL for a browser to become a Spot-Remote.
+     *
+     * @private
+     * @returns {string}
+     */
+    _getSpotRemoteConnectUrl() {
+        return `${windowHandler.getBaseUrl()}/${this.props.joinCode.toUpperCase()}`;
```

Comment:
okay, now the logic for generating the join URL is duplicated in 3 places, is there a way to move this into a helper function or something ?

Reply:
I've moved all the logic into one component. Generally the link creation logic is for debugging and the display of the full URL was never per design. So we'll see what happens when we get more design tweaks.

Reply:
So it looks like in the future the base part won't even exist. But for now we'll keep it,